### PR TITLE
Change message for duplicates

### LINKED ISSUE #298: Duplicate Detection: Error message not helpful enough to users
Let's:
- Let users know that "A student with the same name and the same phone number already exists."

**Note:**
2 contacts with the same name and different phone numbers is still allowed, due to the following cases:
- There are 2 different students with the same name
- The user wants to save 2 phone numbers of the same student but the app only allows one phone number per record. Our aim is to do our best to let the users do what they require.

### PR DESCRIPTION
**Pull request by DennieDan**
Closes #298

### CODE REVIEW

**Review by Soumyadip-cmd — COMMENTED** [review]
The duplicate message in edit command may not have been changed.

**Review by teamotitan — COMMENTED** [review]
LGTM(theres a comment from saha)

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -66,7 +66,6 @@ public class EditCommand extends Command {
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_STUDENT = "This Student already exists in the address book.";
```
[the remark this answers is not in the document]
Comment (DennieDan) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: Thanks for your reminder! Changed already

**Review by Feng1231 — APPROVED** [review]
LGTM

**Pull request merged**